addons/mail/static/src/core/common/picker_content.scss

```diff
@@ -7,3 +7,17 @@
         height: 100% !important;
     }
 }
+
+.o-mail-small-emoji-picker {
+    bottom: 0!important;
+    top:auto !important;
+    left: 0!important;
+
+    .o-mail-PickerContent{
+        width: 100%;
+    }
+
+    .o-Emoji {
+        width: 40px;
+    }
```

Comment:
Instead of redefining style here we could use style from `core/common/picker.scss` . So, it will give consistency for emojipicker in both Discuss App and Chatter.

What you could do in `core/common/picker.scss`
```css
.o-mail-Picker, .o-mail-small-emoji-picker {
    height: 55vh ;
    ...
}
```

And in the current file
```css
.o-mail-PickerContent{
        width: 100%;
        height: 100%;
    }
```